TeamMemberFiles/darien.dalton.txt

```diff
@@ -0,0 +1,5 @@
+"When the fog rolls in, you can fly above it. Never apologize for the strength of your wings. Never pretend you don't love the feeling of flight. The fog will come, we are not fated to fall inside it."
+
+The author of this quote is Dr. Arun Dhir.
```

Comment:
This is great.  When I first say the fog rolls in, I thought it was going to be a quote from the movie "The Perfect Storm"